### PR TITLE
feat: Extend Prefilling with an option to auto-skip prefilled values

### PR DESCRIPTION
**Pull request by gupta-piyush19**
## What does this PR do?
Extend Prefilling with an option to auto-skip prefilled values.

Fixes 
https://github.com/formbricks/internal/issues/145

## How should this be tested?
- Test the functionality of multi-ques prefilling with a new query param `skipPrefilled=1`.
- It should skip the prefilled questions

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [x] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/s/[surveyId]/components/LinkSurvey.tsx [hunk 1]
```diff
@@ -211,6 +212,7 @@ export default function LinkSurvey({
           styling={determineStyling()}
           languageCode={languageCode}
           isBrandingEnabled={product.linkSurveyBranding}
+          isInEditor={false}
```
Comment (pandeymangg) [inline apps/web/app/s/[surveyId]/components/LinkSurvey.tsx:215]: Please rename this prop to something like `isResetProgress` as it conveys better about what this does, and is a boolean

**Review by pandeymangg — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! @gupta-piyush19 😊 
Looks good, just left some comments for minor refactor, pls take a look 🙏

**Review by pandeymangg — APPROVED** [review]
Thanks a lot for the changes! 
LGTM! 🚀

**Review by mattinannt — APPROVED** [review]
@gupta-piyush19 Nice, the feature works great!! :-) 💪
I only changed the value of `skipPrefilling` from `1` to `true` since I noticed we now also use `true` in all other functionality and this felt more consistent with the other features.
Great job; let's merge this :-)

**Pull request merged**